data/sql_partition/partition_schedule_a.sql

```diff
@@ -0,0 +1,259 @@
+    receipt_tp_desc             VARCHAR(90),
+    receipt_desc                VARCHAR(100),
+    memo_cd                     VARCHAR(1),
+    memo_cd_desc                VARCHAR(50),
+    memo_text                   VARCHAR(100),
+    cand_id                     VARCHAR(9),
+    cand_nm                     VARCHAR(90),
+    cand_nm_first               VARCHAR(38),
+    cand_m_nm                   VARCHAR(20),
+    cand_nm_last                VARCHAR(38),
+    cand_prefix                 VARCHAR(10),
+    cand_suffix                 VARCHAR(10),
+    cand_office                 VARCHAR(1),
+    cand_office_desc            VARCHAR(20),
+    cand_office_st              VARCHAR(2),
+    cand_office_st_desc         VARCHAR(20),
+    cand_office_district        VARCHAR(2),
+    conduit_cmte_id             VARCHAR(9),
+    conduit_cmte_nm             VARCHAR(200),
+    conduit_cmte_st1            VARCHAR(34),
+    conduit_cmte_st2            VARCHAR(34),
+    conduit_cmte_city           VARCHAR(30),
+    conduit_cmte_st             VARCHAR(2),
+    conduit_cmte_zip            VARCHAR(9),
+    donor_cmte_nm               VARCHAR(200),
+    national_cmte_nonfed_acct   VARCHAR(9),
+    increased_limit             VARCHAR(1),
+    action_cd                   VARCHAR(1),
+    action_cd_desc              VARCHAR(15),
+    tran_id                     TEXT,
+    back_ref_tran_id            TEXT,
+    back_ref_sched_nm           VARCHAR(8),
+    schedule_type               VARCHAR(2),
+    schedule_type_desc          VARCHAR(90),
+    line_num                    VARCHAR(12),
+    image_num                   VARCHAR(18),
+    file_num                    NUMERIC(7,0),
+    link_id                     NUMERIC(19,0),
+    orig_sub_id                 NUMERIC(19,0),
+    sub_id                      NUMERIC(19,0) NOT NULL,
+    filing_form                 VARCHAR(8) NOT NULL,
+    rpt_tp                      VARCHAR(3),
+    rpt_yr                      NUMERIC(4,0),
+    election_cycle              NUMERIC(4,0),
+    timestamp                   TIMESTAMP,
+    pg_date                     TIMESTAMP,
+    pdf_url                     TEXT,
+    contributor_name_text       TSVECTOR,
+    contributor_employer_text   TSVECTOR,
+    contributor_occupation_text TSVECTOR,
+    is_individual               BOOLEAN,
+    clean_contbr_id             VARCHAR(9),
+    two_year_transaction_period SMALLINT,
+    line_number_label           TEXT
+);
+
+-- Create the child tables.
+SELECT create_itemized_schedule_partition('a', :PARTITION_START_YEAR, :PARTITION_END_YEAR);
+SELECT finalize_itemized_schedule_a_tables(:PARTITION_START_YEAR, :PARTITION_END_YEAR);
+
+-- Create the insert trigger so that records go into the proper child table.
+DROP TRIGGER IF EXISTS insert_sched_a_trigger_tmp ON ofec_sched_a_master_tmp;
+CREATE trigger insert_sched_a_trigger_tmp BEFORE INSERT ON ofec_sched_a_master_tmp FOR EACH ROW EXECUTE PROCEDURE insert_sched_master('ofec_sched_a_');
```

Comment:
Shouldn't we do the full insert into the tables before the indexes are created to help speed up inserting the records into the partition and ensure the indexes are fresh with the new data?